.circleci/config.yml

```diff
@@ -1181,6 +1267,32 @@ workflows:
               catch2_version:
                 - "2.13.4"
 
+  zend_abstract_interface:
```

Comment:
I suggest to rename it `zai` as a long name would hide all the rest in the CircleCI UI, and you have to stay over to have a full description.

Reply:
Is there a particular spot in the UI where a long workflow name truncates the job name? I can't seem to find it. I named the job `Zai` for the same reason you're mentioning and so far it's looking pretty good in the UI.

<img width="404" alt="Screen Shot 2021-04-15 at 9 38 22 AM" src="https://user-images.githubusercontent.com/578780/114891017-859c2900-9dd9-11eb-8a78-e6ce47437309.png">

<img width="733" alt="Screen Shot 2021-04-15 at 9 39 24 AM" src="https://user-images.githubusercontent.com/578780/114891019-8634bf80-9dd9-11eb-8c8e-897ef37c89ae.png">

But are you referring to a different spot?

Reply:
I had missed the notification about this message, so I am answering late. For some reason, I thought that this was the job definition, while it is the workflow name (which is not subject to the issue I mentioned). Thanks for looking though it, though.